add kaiserschmarrn recipe (#934)
LukeSmithxyz/based.cooking: content/kaiserschmarrn.md

---
title: Kaiserschmarrn
tags: ['sweets', 'sweet', 'german', 'austrian']
date: 2023-04-30
author: "Felix Steiner"
---

> A very recoginzable and simple sweet dish from the mountainous regions of Austria, souther Germany and northern Italy.

Reviewer: ```suggestion
A very recognizable and simple sweet dish from the mountainous regions of Austria, southern Germany and northern Italy.
```

- ⏲️ Prep time: 10 min
- 🍳 Cook time: 10 min
- 🍽️ Servings: 3

## Ingredients

> - 200g plain flower

Reviewer: what is "plain" flour? i guess you mean type-405 flour, which would be "pastry flour" [for americans](https://ellerepublic.de/en/german-vs-american-flour/)?

```suggestion
- 200g wheat flour (type 405 / pastry flour)
```

- 30g sugar
- 4 eggs
- 300ml milk
- 40g butter
- 30g raisins (optional)

## Directions

1. Separate the egg whites and yolks.

> 2. Put the egg yolks, flour, sugar, pinch of salt, and milk in a bowl and stir until the clumps are gone

Reviewer: ```suggestion
2. Put the egg yolks, flour, sugar, pinch of salt, and milk in a bowl and stir until the clumps are gone.
```

> 3. Beat the egg whites to stiff peaks in a different bowl and then add to the other bowl

Reviewer: ```suggestion
3. Beat the egg whites to stiff peaks in a different bowl and then add to the other bowl.
```

4. Melt the butter in a flat pan on high heat and then add the dough. Heat up on both sides.

> 5. Put the pan in the preheated oven (180°C hot air setting) for around 10 minutes or until lightly browned

Reviewer: ```suggestion
5. Put the pan in the preheated oven (180°C hot air setting) for around 10 minutes or until lightly browned.
```

> 6. Tear up the dough with a spatula into bite-sized pieces.

Reviewer: ```suggestion
6. Cut up the dough with a spatula into bite-sized pieces.
```

I think "tearing" gives the wrong impression. Something you do with your hands, explicitly away from each other.

> 7. Sprinkle some sugar on top (add raisins here)

Reviewer: ```suggestion
7. Sprinkle some sugar on top (add raisins here).
```
